app/scrapers/__init__.py

```diff
@@ -42,9 +42,9 @@ def feed_gen(query, engine, count=10, qtype=''):
     engine = old_names.get(engine, engine)
     if engine in ('quora', 'youtube'):
         urls = scrapers[engine].search_without_count(query)
-    elif (engine in ['parsijoo']) and (qtype == 'isch'):
+    elif engine in ('parsijoo') and qtype == 'isch':
```

Comment:
__('parsijoo')__ is not a tuple, it is a str.  __('parsijoo', )__ is a tuple.  However the test still works as expected.

$ __python3 -c "print(type(('parsijoo')), type(('parsijoo', )))"__